java/arcs/android/sdk/host/AndroidHost.kt

```diff
@@ -32,30 +31,15 @@ abstract class AndroidHost(
     val context: Context,
     lifecycle: Lifecycle,
     schedulerProvider: SchedulerProvider,
-    override val activationFactory: ActivationFactory,
     vararg particles: ParticleRegistration
 ) : JvmHost(schedulerProvider, *particles), DefaultLifecycleObserver {
 
-    @ExperimentalCoroutinesApi
-    constructor(
-        context: Context,
-        lifecycle: Lifecycle,
-        schedulerProvider: SchedulerProvider,
-        vararg particles: ParticleRegistration
-    ) : this(
-        context,
-        lifecycle,
-        schedulerProvider,
-        ServiceStoreFactory(context),
-        *particles
-    )
-
     init {
         lifecycle.addObserver(this)
     }
 
     @ExperimentalCoroutinesApi
-    override val stores: StoreManager = StoreManager(activationFactory)
+    override val stores: StoreManager = StoreManager(ServiceStoreFactory(context))
```

Comment:
I think we may want to keep the activationFactory injectable via constructor rather than hardcoding it here. Wouldn't it need to be configurable by clients who extend the base StorageService?

Reply:
I can put it back, but no one is using it at the moment.

My goal is to eventually remove this class in favor of providing configuration parameters (right now, just coroutine scopes and a HandleManagerProvider).

Reply:
I changed this so that AndroidHost takes a `StoreManager` argument, which is more the direction this will go (eventually, this will look more like `BaseArcHost(handleManager = AndroidHandleManagerProvider, etc)`

Reply:
Aaaand that broke something else downstream :( I'm going to undo this for now, sorry. 

It's an open val, so it's still overridable.

Reply:
Ok, you win , because there was a piece of code expecting an overridable `activationFactory` :-D